src/bot/setupEvent.ts

```diff
@@ -40,27 +43,47 @@ export const setupEvent = <E extends WebhookEvents>(
             body: `${comment.requester ? `@${comment.requester} ` : ""}${comment.body}`,
           };
 
+          counters.commandsWarn.inc({
```

Comment:
Why is this a warning?

Reply:
PullRequestError class might sound confusing here, but it's expected response if somebody used bot, but incorrectly, so it will respond with recommendation how to fix or go to docs
Next line is eventLogger.warn too